### ISSUE TITLE
(Feature request) Activate tab if opened via `Open in a new tab` context menu item

### ISSUE THREAD

**Issue by meichthys** [TriliumNext/Trilium#3970]
### Describe feature

Currently when opening a note in a new tab, the note tab is appended to the end of the tab list in the active editor. This is probably what would be expected when middle clicking a note to open it, or when `ctrl-click`ing it, but when using the context menu, it seems like the note itself should be activated after it is added to the editor. 

When a user goes through the (albeit small) effort to use the `Open in a new tab` cotext menu, they are most likely looking to view that note at the same time (perhaps i could be wrong on this).

**Comment by zadam** [comment]
In browsers, "Open in a new tab" does not activate the tab even if triggered from a context menu.

We could introduce a possibility to configure new tab behavior if there's enough demand, but to me, it seems like users should be used to tab opening without activation from browsers.

**Comment by soulsands** [comment]
Personally, I prefer to activate a newly opened tab directly, regardless of how it is opened. I can't think of a scenario where I would only want it to open in the background. I would like to see the content immediately everytime I open a note. I think it's best to make it configueable.

**Comment by harveyTon** [comment]
This is also what I want

**Comment by tevenfeng** [comment]
> In browsers, "Open in a new tab" does not activate the tab even if triggered from a context menu.
well, in browsers people are often thought to be reading, when they open a new tab, we may assume that they will activate the newly opened tab when they finish reading the current tab.
However, in Trilium we open a new tab because we want to edit it or to get something from it immediately. That's the most common scenario when using a note app, different from using browsers.

**Comment by medblg** [comment]
Exactly, it's annoying when creating a new note and it's replacing the current note tab, as @tevenfeng stated, the context is different when using a note app.

btw thanks for the great app.

**Comment by SiriusXT** [comment]
Currently, when pressing Ctrl+Shift+Click, the note opens in a new tab and is automatically activated, consistent with the browser’s default behavior.
